source/docs/12-facsimilesrecordings.xml

```diff
@@ -299,6 +299,14 @@
                      </figure>
                   </p>
                   <p>indicates that the entities identified in <att>data</att> all occur at the same instant.</p>
+                  <p><att>extData</att> is a container for holding non-MEI data formats, similar to <att>extMeta</att> but available in <att>when</att> rather than in MEI.header. <att>extData</att> allows for data from audio or other sources to be linked to notes or other score events. Data should be enclosed in a CDATA tag.</p>
+                  <p>The following example shows JSON formatted performance data encoded with <att>extMeta</att> for a single note (presumed to be defined elsewhere in the document as with the ID "note_1"). Both single-value summaries (e.g., pitch) and time series values (e.g., contF0) are encoded. </p>
```

Comment:
<p>The following example shows JSON formatted performance data encoded with <gi>extMeta</gi> for a single note (presumed to be defined elsewhere in the document as with the ID "note_1"). Both single-value summaries (e.g., pitch) and time series values (e.g., contF0) are encoded. </p>